CMakeLists.txt

```diff
@@ -68,6 +68,11 @@ cpr_option(CPR_BUILD_TESTS_SSL "Set to ON to build cpr ssl tests" ${CPR_BUILD_TE
 cpr_option(CPR_BUILD_TESTS_PROXY "Set to ON to build proxy tests. They fail in case there is no valid proxy server available in proxy_tests.cpp" OFF)
 cpr_option(CPR_SKIP_CA_BUNDLE_SEARCH "Skip searching for Certificate Authority certs. Turn ON systems like iOS where file access is restricted and prevents https from working." OFF)
 cpr_option(CPR_USE_BOOST_FILESYSTEM "Set to ON to use the Boost.Filesystem library on OSX." OFF)
+cpr_option(CPR_DEBUG_SANITIZER_FLAG_THREAD "Enable ThreadSanitizer for debug build." OFF)
+cpr_option(CPR_DEBUG_SANITIZER_FLAG_ADDR "Enable AddressSanitizer for debug build." OFF)
+cpr_option(CPR_DEBUG_SANITIZER_FLAG_LEAK "Enable LeakSanitizer for debug build." OFF)
+cpr_option(CPR_DEBUG_SANITIZER_FLAG_UB "Enable UndefinedBehaviorSanitizer for debug build." OFF)
+cpr_option(CPR_DEBUG_SANITIZER_FLAG_ALL "Enable all sanitizers for debug build." OFF)
```

Comment:
```suggestion
cpr_option(CPR_DEBUG_SANITIZER_FLAG_ALL "Enables all sanitizers for debug builds except the ThreadSanitizer since it is incompatible with the other sanitizers." OFF)
```